ament_cpplint/ament_cpplint/cpplint.py

```diff
@@ -5332,28 +5343,12 @@ def CheckLanguage(filename, clean_lines, linenum, file_extension,
           'Did you mean "memset(%s, 0, %s)"?'
           % (match.group(1), match.group(2)))
 
-  # Check for 'using namespace' which pollutes namespaces.
```

Comment:
Hmmm. It seems this was another un-documented patch see https://github.com/ament/ament_lint/pull/67. One the one hand it has rejected by the upstream google maintainers however, we could maybe try again with the cpplint fork. On the other hand it seemed to be more of work around of [this](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65923) gcc bug that has been fixed for some time now.

Reply:
I would re-apply the patch just to avoid all those linter errors. We can decide whether to drop it (and fix downstream packages) separately.